packages/grid_client/docs/architecture/decisions/update_deployments_store_path.md

```diff
@@ -0,0 +1,21 @@
+# 2. update_deployment_store_path
+
+Date: 2023-10-22
+
+## Status
+
+Done
+
+## Context
+
+Previously we had no way to group folders so we decided to allow combining related contracts as a folder structure can be used in different project (e.g playground).
+
+## Decision
+
+- We are migrating old key to a new one which allows us to combine related contracts together as a folder grouping.
+- Pattern: `[network]/[twinId]/[projectName]/[moduleName]/[instanceName]/contracts.json` to `[network]/[twinId]/[moduleName]/[projectName]/[instanceName]/contracts.json`
```

Comment:
This doesn't shows as a hierarchy in preview as in packages/playground/docs/decisions/update_deployments_store_path.md. Please adjust.